_episodes/00-why-hpc.md

```diff
@@ -1,13 +1,96 @@
 ---
 title: "Introducing High-Performance Computing"
 questions:
-- "Why use a large-scale computing system?"
+- "What is a super computer?"
```

Comment:
I see "super computer" used here and "High Performance computer" used in another related PR and I suspect there is even other terminology used elsewhere. I would avoid "super computer". I think it is too vague to be a really useful descriptor for the kind of system this material is intended to support. In any event, among all these lesson sections there of course needs to be some consensus on terminology.